cli/mender-artifact/artifacts.go

```diff
@@ -159,10 +158,16 @@ func unpackArtifact(name string) (string, error) {
 		return "", errors.Wrap(err, "failed to register install handler")
 	}
 
-	err = aReader.ReadArtifact()
+	r, err := read(aReader, nil, nil)
 	if err != nil {
 		return "", err
 	}
+
+	inst := r.GetHandlers()
```

Comment:
Maybe `getUpdateType` would later prove handy as a function call?

Reply:
I missed this comment. But I guess it is okey to leave it for the next refactoring day... ;)